src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -72,6 +73,9 @@ object Boot extends App with LazyLogging {
 
     for {
       _ <- schemaDAO.init() recover {
+        case e: WorkbenchException =>
+          logger.error("FATAL - could not update schema to latest version. Is the schema lock stuck? See documentation here for more information: [link]")
```

Comment:
was thinking of making a broad.io link and having it link to some confluence page with instructions

Reply:
It would be great if we could reduce it to a `docker run` command that could be logged with this error.

Reply:
This could also be useful to know during development so we can remove version locks so sam will re-run the schema update without having to artificially increment the version.